app/controllers/home_controller.rb

```diff
@@ -0,0 +1,7 @@
+class HomeController < ApplicationController
+  def index
+    # if user_signed_in?
+    #  redirect_to groups_path
+    # end
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻